### PR TITLE
macbookpro 3,1: init

### PR DESCRIPTION
**Pull request by Krutonium**
###### Description of changes
Added what's needed to make the MacBook Pro 3,1 17" that I have work. (If anyone happens to know how to make Nouveau agree with this device, I'd love to know.)

###### Things done

- [x] Tested the changes in your own NixOS Configuration
- [ ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by Krutonium** [comment]
....And I just realized I edited the wrong files.

**Comment by Krutonium** [comment]
(if anyone does happen to be or know someone who is good with nouveau issues, hmu)

**Comment by Krutonium** [comment]
So I chatted with some people in the nouveau IRC, I was wondering what the right thing to do would be if the solution, realistically, was to distribute a copy of the VBIOS, since it's difficult to extract, and the GPU is non functional without it.

File: apple/macbook-pro/3-1/README.wiki [hunk 1]
```diff
@@ -0,0 +1,5 @@
+= Apple MacBook Pro 3,1 =
```
Comment (Mic92) [inline apple/macbook-pro/3-1/README.wiki:1]: Can you also add this to the README and the flake.nix as described in https://github.com/NixOS/nixos-hardware/blob/master/CONTRIBUTING.md#2-adding-entry ?

**Comment by Mic92** [comment]
> So I chatted with some people in the nouveau IRC, I was wondering what the right thing to do would be if the solution, realistically, was to distribute a copy of the VBIOS, since it's difficult to extract, and the GPU is non functional without it.

There are some places like archive.org where these things can be uploaded... or some throw-away github account in a github repository release. This is a grey area surely but legal risks are not to high for older hardware.

**Comment by toastal** [comment]
> 3,1

Why not 3.1?

**Comment by Mic92** [comment]
Please reopen if you think this is ready.

**Pull request closed without merging**